metatron-app/metatron-discovery
# F#2496 show message for error case

### Description
When user uploads a wrong formatted file, 
error message should be shown. about unsupported file or invalid format

and
There are included issues because there is work on the same area of ​​the same page.
(the page of uploading lineage is made according to the planing document.)

**Related Issue** :
[#2496 ](https://github.com/metatron-app/metatron-discovery/issues/2496) 
METATRON-2995
METATRON-2998
METATRON-2999
METATRON-3001

### How Has This Been Tested?
1. upload an unsupported file like .json
2. upload a file has invalid format. (has no metaId and metaName)
error message should be shown

and the pages will be shown as planning

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
